src/System.Net.Requests/src/System/Net/FtpDataStream.cs

```diff
@@ -56,6 +56,8 @@ protected override void Dispose(bool disposing)
             }
         }
 
+        //TODO: Add this to FxCopBaseline.cs once https://github.com/dotnet/roslyn/issues/15728 is fixed
```

Comment:
I wasn't looking closely at all of the usages. This change is just about getting all of the baselines in. Then we can fix the individual issues and remove the suppression.